src/components/index.ts

```diff
@@ -1,5 +1,6 @@
 export {ActionBar} from './ActionBar';
 export {AsideHeader, AsideHeaderProps} from './AsideHeader/AsideHeader';
+export {CompositeBar, CompositeBarProps} from './CompositeBar/CompositeBar';
```

Comment:
What problem do you want to solve? These are internal components and are not intended to be used externally.

Reply:
@DarkGenius hello. I want to customize expand button in AsideHeader component, and main purpose is - display expand button with custom icon and custom styles (like menu items in aside header).   

But currently I can not do that, and exporting composite bar and customize it will be good solution for me.

Reply:
@js2me Hi! The ability to customize the expand button should be implemented by adding new properties to the AsideHeader component. You can create an appropriate issue for this.